sci-libs/rocBLAS/rocBLAS-4.1.0.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="AMD's library for BLAS on ROCm."
+HOMEPAGE="https://github.com/ROCmSoftwarePlatform/rocBLAS"
+SRC_URI="https://github.com/ROCmSoftwarePlatform/rocBLAS/archive/rocm-${PV}.tar.gz -> rocm-${PN}-${PV}.tar.gz
+	https://github.com/ROCmSoftwarePlatform/Tensile/archive/rocm-${PV}.tar.gz -> rocm-Tensile-${PV}.tar.gz"
+
+LICENSE="MIT"
+KEYWORDS="~amd64"
+IUSE="benchmark test"
+SLOT="0"
+
+RDEPEND="=dev-util/hip-$(ver_cut 1-2)*"
+DEPEND="${RDEPEND}
+	dev-perl/File-Which
+	dev-python/pyyaml
```

Comment:
This goes to `BDEPEND`

Reply:
```
     $(python_gen_any_dep '
         dev-python/polib[${PYTHON_USEDEP}]
         dev-python/pygobject:3[${PYTHON_USEDEP}]
     ')

```
python-any-r1 dependencies require the above calls in BDEPEND.

Reply:
rocBLAS depend hip (libamdhip64.so) at runtime

Reply:
> rocBLAS depend hip (libamdhip64.so) at runtime

I meant only `dev-python/pyyaml`.